Nishant/postgres assignment/src/controller/student.ts

```diff
@@ -0,0 +1,73 @@
+import { Context } from 'vm';
+import { QueryResult } from 'pg';
+import * as studentService from '../services/student';
+import addStudentSchema from '../helper/studentvalidation';
+
+interface IStudent{
+  student_id: number;
+  fname:string;
+  mname:string;
+  lname:string;
+  dob :Date;
+  gender: string;
+  address:string;
+  slice(a:number, b:number): IStudent;
+}
+
+export async function getStudent(ctx: Context) {
+  try {
+    const rows:QueryResult = await studentService.getStudentService();
```

Comment:
you can use exception handling only for those statements that would raise exception... keeping all the statements inside try/catch would not tell me immediately as to what will raise exception...